### ISSUE TITLE
Adding a template feature to okteto init

### ISSUE THREAD

**Issue by zachhutchins** [okteto/okteto#4314]
### Discussed in https://github.com/okteto/okteto/discussions/4296

<div type='discussions-op-text'>

<sup>Originally posted by **zachhutchins** May 17, 2024</sup>
One thing I do not like about `okteto init` is the lack of customization options. 

Thinking of adding `okteto init --template <file|url> --arg <name>=<value>[]`

Any major objections?

P.S Contribution guide says to communicate changes first

# Example

### Using a local file as a template with additional arguments
okteto init --template ./path/to/template.yaml --arg "config=dev" --arg "debug=true"

----

### Using a URL as a template
okteto init --template https://example.com/template.yaml --arg "branch=feature1"</div>

**Comment by codyjlandstrom** [comment]
Hi @zachhutchins 
Apologies for the delay here. We're currently planning on deprecating `okteto init` as it creates some confusion with the v2 of the manifest.
Could you describe how you're using `okteto init` now and if this would be a large disruption to your workflow?

**Comment by zachhutchins** [comment]
@codyjlandstrom Currently I am not because it just does not fit my workflow currently, I ended up forking and making the template feature because the manifests I use have a lot of lengthy annotations and other aspects of the manifest I need to add for our environment. 

We also have some devs who are just not really privy to Kubernetes or okteto manifests and so a template with a couple of arguments works perfect to get them off the ground and running quick.

I found a template to work best for me, if you are interested, I would love to introduce my feature. I would be interested If you are removing the `init` command what are your plans to replace it with.
